compiler/src/iree/compiler/Dialect/Util/Transforms/Passes.td

```diff
@@ -105,6 +105,11 @@ def HoistIntoGlobals : Pass<"iree-util-hoist-into-globals", "mlir::ModuleOp"> {
   let constructor = [{
     mlir::iree_compiler::IREE::Util::createHoistIntoGlobalsPass()
   }];
+  let options = [
+    Option<"maxSizeIncreaseThreshold", "max-size-increase-threshold", "int64_t",
+      /*default=*/"2147483647",
```

Comment:
Why is the default value this instead of 1024 * 1024 as before? We chose 1024 * 1024 to prevent const eval of anything more than 1MB by default.

Reply:
Thanks for the explanation, I was confused why it's 1024x1024.